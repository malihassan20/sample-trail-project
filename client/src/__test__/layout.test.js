import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import Layout from '../components/layout';

test('Layout component renders without crashing', () => {
  render(<Layout />);
});

it('Layout component matches the snapshot', () => {
  const tree = renderer.create(<Layout />).toJSON();
  expect(tree).toMatchSnapshot();
});
