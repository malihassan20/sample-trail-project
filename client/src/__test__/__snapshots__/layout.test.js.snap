// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component matches the snapshot 1`] = `
<div
  className="container mt-4 pt-4 mb-4 pb-4 pl-4 pr-4"
/>
`;
